### ISSUE TITLE
Buttons with save and cancel option

### ISSUE THREAD

**Issue by akogo1** [e107inc/e107#3913]
Is this a js scripts conflict? On each custom page appear buttons with save and cancel option. Links looks viewable but overlapped.

**Comment by tgtje** [comment]
Confirmed.; more info at http://e107help.org/4672/buttons-with-save-and-cancel-option

**Comment by akogo1** [comment]
Links on every custom page don't work properly. Where can I find a solution to this bug?

**Comment by Jimmi08** [comment]
It's because of sidebar.  This functionality was probably planned/tested with full page layout.

Try this:
```
#e-editable-front-controls{
 z-index: 9999;
}
```

**Comment by tgtje** [comment]
Tested it on BS3 (no result with above code).
But now point of reference >
Looking at web/e107.css noticed (small screen)

> .e-editable-front { position:relative }
.e-editable-front:hover,
.e-editable-front a:hover { outline: 1px dashed red; }
.e-editable-front.active {  -moz-box-shadow: inset 0 0 5px 2px #888;
-webkit-box-shadow: inset 0 0 5px 2px#888;
box-shadow: inset 0 0 5px 2px #888; }
**#e-**editable-front-controls { position: absolute;
Now _nearly_ all surrounding…. has a line started with a dot ( . ), while the marked line uses a #.
Changing this into a dot too, i get result.. Maybe there lies the issue?
![edit3](https://user-images.githubusercontent.com/1811857/63649126-f771bb80-c739-11e9-98f3-a3805e402fab.jpg)

**Comment by akogo1** [comment]
Thank you, but no changes :/
In the news title link too...

![newsbug](https://user-images.githubusercontent.com/3391851/63655866-a390c200-c78d-11e9-90aa-a2d724dfe228.png)

**Comment by tgtje** [comment]
Ok see it. When in Frontend Inline-Editing: mode=on, the setting overrides the linkage.
Disabling frontend edit gives link back.

So there is an issue... the display would be ok, but the override has to be looked into...(point of discussion if and who should be able to front editing...(rights/class to change)

(ps cache emptied needed after changing the css)

**Comment by akogo1** [comment]
Yes, after disabling Frontend Inline-Editing (No one) the links work!
